modules/installation-and-upgrade/partials/container-deployment/generate_proxy_config.adoc

```diff
@@ -22,6 +22,27 @@ For Podman deployment, the container host for the {productname} Proxy must be re
 If a proxy FQDN is used to generate a proxy container configuration that is not a registered client (as in the Kubernetes use case), a new system entry will appear in system list.
 This new entry will be shown under previously entered Proxy FQDN value and will be of [literal]``Foreign`` system type.
 
+[NOTE]
+====
+Peripheral servers are always using third-party SSL certificates.
+If the hub server has generated the certificates for the peripheral server, it needs to generate the certificate of each proxy too.
+On the hub server, run the following command.
+[source,shell]
+----
+mgrctl exec -ti -- rhn-ssl-tool --gen-server --dir="/root/ssl-build" --set-country="COUNTRY" \
```

Comment:
Isn't this command `-it` (interactive terminal)?

Reply:
It's -ti elsewhere in docs.